main.cpp

```diff
@@ -353,10 +420,26 @@ struct Wind
     std::string definition = "movement in air molecules";
     std::string cause = "difference in gas density";
     std::string use;
+    
 
     void transportSeeds (int, std::string, int);
     bool turnTurbines (std::string, float gust = 34.2f);
     void helpBirdTravel (std::string direction2, bool birdFly, float gust = 34.2f);
+    void beaufortScale(int gust, int highParam)
+    {
+        int lowerParam = 1;
+        while (highParam <= 64)
+        {
+            if (gust > lowerParam  && gust < highParam )
+            {
+                lowerParam += 6;
+                std::cout << "Within Beaufort scale." << std::endl;
+            }
+            std::cout << "Gust: " << gust << std::endl;
+            break;
+        }
```

Comment:
remove the `break`, because it prevents the `while` loop from looping.
- [ ] revise